frontend/src/component/feedbackNew/FeedbackComponent.tsx

```diff
@@ -1,7 +1,12 @@
-import { Box, Button, styled, TextField } from '@mui/material';
+import { Box, Button, IconButton, styled, TextField, Tooltip } from '@mui/material';
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 import { useFeedback } from './useFeedback';
-import React from 'react';
+import React, { useState } from 'react';
+import CloseIcon from '@mui/icons-material/Close';
+import produce from 'immer';
```

Comment:
are you sure you want to add immer for such a simple use case?

Reply:
Removed